public/templates/feeds.html

```diff
@@ -15,7 +15,7 @@
 
     <li ng-class="{ 'bold': feedType === 'top' }">
       <a
-        ng-href="{{ hashtag ? '/hashtag/' + hashtag : '' }}/top"
+        ng-href="{{ hashtag ? '/hashtag/' + hashtag : '' }}/top/last-month"
```

Comment:
We cannot change the links because they have already been indexed by SEO. It should be something like this:
/top <- displays one month by default
/top?type=all-time

Reply:
sure